DEVELOPING.md

```diff
@@ -0,0 +1,22 @@
+# Developing
+
+## Quick Start
+
+FIRST, install the OCaml dependencies and generate the OCaml -> Javascript
+artifacts:
+
+```bash
+opam install gen_js_api js_of_ocaml jsonoo opam-file-format promise_jsoo
```

Comment:
The dependencies can automatically be read from the opam files (like how it's done in CI).
```suggestion
opam install . --deps-only --with-test
```